app/controllers/posts_controller.rb

```diff
@@ -4,11 +4,11 @@ class PostsController < ApplicationController
 
   def index
     @posts = Post.all
-    render json: @posts
+    render 'index'
```

Comment:
[OPTIONAL]
- [ ] Kindly note that rails automatically render the view that have the same name as the action in the controller. Therefore there is no need to add the `render 'index'` to the `index` action, rails will render `app/views/posts/index.html.erb` by default for the `index` action in the posts controller.

Reply:
👍